holoviews/tests/ui/bokeh/test_layout.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+import pytest
+
+pytestmark = pytest.mark.ui
```

Comment:
There's a warning about this line in the logs. There's more code that should be added to `conftest.py` maybe? https://github.com/holoviz/panel/blob/60ba9c91990044e8f06c3cc9ca2bc66c4dab73b1/panel/tests/conftest.py#L81

Links to and copy of the warning:
https://github.com/holoviz/holoviews/actions/runs/5290689182/jobs/9575200902?pr=5764#step:4:288

```python
holoviews/tests/ui/bokeh/test_layout.py:5
  /home/runner/work/holoviews/holoviews/holoviews/tests/ui/bokeh/test_layout.py:5: PytestUnknownMarkWarning: Unknown pytest.mark.ui - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    pytestmark = pytest.mark.ui
```